### PR TITLE
Add upgraded laboratory and storage (and adds upgraded buildings to holding)

### PR DESCRIPTION
**Pull request by sagarg22**
I renamed the 'laboratory' buildings to 'lab' and 'storage' to 'storehouse' so that upgraded buildings with proper naming conventions/types could be added.
screenshot 13
### **^Upgraded storage**
screenshot 14
screenshot 15
### **^Inside of upgraded lab**
screenshot 16

### CODE REVIEW

**Comment by namtiw** [comment]
Great job! The new building and its upgrade work correctly.
But there are still a few bugs to address:
- The Laboratory is added to the holding after the upgrade, but the old instance is not removed. So the count on the overview screen becomes erroneous.
- After the upgrade, the Laboratory is not correctly "initialized" i.e. its Pedestal block is not registered. So, after the upgrade a research task cannot be added from this Laboratory.

File: assets/prefabs/structureTemplates/types/laboratory1.prefab [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "StructureTemplateType" : {}
```
Comment (skaldarnar) [inline assets/prefabs/structureTemplates/types/laboratory1.prefab:2]: File should be renamed to `laboratoryLevel1` according to existing naming schemes.

File: assets/prefabs/structureTemplates/types/storage1.prefab [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "StructureTemplateType" : {}
```
Comment (skaldarnar) [inline assets/prefabs/structureTemplates/types/storage1.prefab:2]: File should be renamed to `storageLevel1` according to existing naming schemes.

**Review by skaldarnar — CHANGES_REQUESTED** [review]
I'll merge this into a new branch and apply the required changes.

**Pull request closed without merging**